ngraph/frontend/onnx_import/include/onnx_import/core/tensor.hpp

```diff
@@ -171,6 +172,15 @@ namespace ngraph
                 template <>
                 inline std::vector<double> get_data(const ONNX_NAMESPACE::TensorProto& tensor)
                 {
+                    if (tensor.has_data_location() &&
+                        tensor.data_location() == ONNX_NAMESPACE::TensorProto_DataLocation::
+                                                      TensorProto_DataLocation_EXTERNAL)
+                    {
+                        const auto external_data_info = ExternalDataInfo(tensor);
```

Comment:
The 3 lines inside this if statement are repeated for each type. Can we put them in some kind of a template function like `template<typename T> std::vector<T> get_external_data(TensorProto)`?

Reply:
right, done